apps/developer-hub/content/docs/price-feeds/hip-3-service/index.mdx

```diff
@@ -14,26 +14,26 @@ Pyth Network provides **HIP-3 as a Service**, a complete end-to-end solution for
 
 HIP-3 as a Service provides everything you need to launch and maintain a successful perpetual market on Hyperliquid:
 
-<div className="grid grid-cols-1 md:grid-cols-3 gap-6 mt-8">
+<div className="grid grid-cols-1 gap-4 sm:grid-cols-2 lg:grid-cols-3">
```

Comment:
Can we replace tailwind and use style sheet? Like this
https://github.com/pyth-network/pyth-crosschain/blob/main/apps/developer-hub/src/components/Pages/Homepage/index.module.scss